.github/workflows/ubuntu.yml

```diff
@@ -36,8 +36,6 @@ jobs:
           - test_task: test-bundler-parallel
             timeout: 50
           - test_task: test-bundled-gems
-          - test_task: check
-            os: ubuntu-20.04
```

Comment:
Why not test on ubuntu-22.04?

Reply:
We tested `make check` with default `runs-on` variable like `runs-on: ${{ matrix.os || 'ubuntu-22.04' }}`.